src/backend/distributed/utils/reference_table_utils.c

```diff
@@ -382,13 +382,13 @@ CreateReferenceTableColocationId()
 
 
 /*
- * DeleteAllReferenceTablePlacementsFromNode function iterates over list of reference
+ * DeleteAllReferenceTablePlacementsFromGroup function iterates over list of reference
  * tables and deletes all reference table placements from pg_dist_placement table
- * for given worker node. However, it does not modify replication factor of the colocation
+ * for given group. However, it does not modify replication factor of the colocation
  * group of reference tables. It is caller's responsibility to do that if it is necessary.
  */
 void
-DeleteAllReferenceTablePlacementsFromNode(char *workerName, uint32 workerPort)
+DeleteAllReferenceTablePlacementsFromGroup(uint32 groupId)
```

Comment:
We don't have groups of anything other than nodes yet, but group is such a generic term that it might confuse new readers.

I think maybe `GroupHasShardPlacements` would be slightly more clear as `NodeGroupHasShardPlacements`.